_episodes/02-meet-docker.md

```diff
@@ -8,52 +8,60 @@ objectives:
 
-The Docker application will usually provide a way for you to log in using the application's menu (macOS) or systray icon (Windows). This will require you to use your Docker Hub username and your password.
+> ## You may need to login to Dockerhub
+> The Docker application will usually provide a way for you to log in to the Dockerhub using the application's menu (macOS) or systray
+> icon (Windows) and it is usually convenient to do this when the application starts. This will require you to use your Docker Hub
+> username and your password. We will not actually require access to Dockerhub until later in the course but if you can login now,
+> you should do so.
+{: .callout}
 
 > ## Determining your Docker Hub username
-> If you no longer recall your Docker Hub username, e.g., because you have been logging into the Docker Hub using your email address, you can find out what it is through the steps:
+> If you no longer recall your Docker Hub username, e.g., because you have been logging into the Docker Hub using your email address,
+> you can find out what it is through the steps:
 > - Open <http://hub.docker.com/> in a web browser window
 > - Sign-in using your email and password (don't tell us what it is)
-> - In the top-right of the screen you will see your username.
+> - In the top-right of the screen you will see your username
 {: .callout}
 
-Now open a shell window, and run the following command in your shell to check that Docker is installed. I have appended the output that I see on my Mac, but the specific version is unlikely to matter much: it certainly does not have to precisely match mine.
+Once your Docker application is running, open a shell (terminal) window, and run the following command to check that Docker is
+installed and the command line tools are working correctly. I have appended the output that I see on my Mac, but the specific
+version is unlikely to matter much: it certainly does not have to precisely match mine.
 ~~~
 $ docker --version
 ~~~
 {: .language-bash}
 ~~~
-Docker version 18.09.1, build 4c52b90
+Docker version 19.03.5, build 633a0ea
 ~~~
 {: .output}
 
-Ensure that your command line `docker` commands are able to reach the Docker Hub by running the following command:
+The above command has not actually relied on the part of Docker that runs lightweight virtual machines being operational. Somewhat stretching a physical analogy, you can think of the above Docker command having been instructions to the cranes on a hypothetical shipping dock, but we haven't actually checked if the container ship we want to interact with is present yet. A command that checks that the virtual machine host is running is the Docker container list command (we cover this command in more detail later in the course).
```

Comment:
I find this a bit confusing.  Could we simplify it a bit more?

We should also probably break it up into multiple lines.